pkg/cfn/manager/tasks.go

```diff
@@ -65,23 +67,27 @@ func (c *StackCollection) CreateClusterWithNodeGroups() []error {
 		return errs
 	}
 
-	return c.CreateAllNodeGroups()
+	return c.CreateAllNodeGroups(nil)
 }
 
 // CreateAllNodeGroups runs all tasks required to create the node groups;
 // any errors will be returned as a slice as soon as one of the tasks
 // or group of tasks is completed
-func (c *StackCollection) CreateAllNodeGroups() []error {
+func (c *StackCollection) CreateAllNodeGroups(onlySubset sets.String) []error {
 	errs := []error{}
 	appendErr := func(err error) {
```

Comment:
As far as I recall, we do it this way because we want to collect errors from multiple parallel tasks. There may be another way to do it, I've not looked into this in a long time.